src/utils/validateNric.test.tsx

```diff
@@ -2,12 +2,16 @@ import { validateAndCleanNric } from "./validateNric";
 
 describe("validateAndCleanNric", () => {
   it("should return cleaned nric for valid nric number", () => {
-    expect.assertions(10);
+    expect.assertions(14);
     // https://en.wikipedia.org/wiki/National_Registration_Identity_Card
     expect(validateAndCleanNric("S0000001I")).toBe("S0000001I");
     expect(validateAndCleanNric("S0000002G")).toBe("S0000002G");
     expect(validateAndCleanNric("S0000003E")).toBe("S0000003E");
     expect(validateAndCleanNric("S0000004C")).toBe("S0000004C");
+    expect(validateAndCleanNric("M1234567K")).toBe("M1234567K");
```

Comment:
I noticed we don't have tests for NRICS starting with T, F, and G, can we add them?